en/docs/reference/config-catalog.md

```diff
@@ -921,6 +921,108 @@ claims_extractor_impl = "org.wso2.carbon.apimgt.impl.token.DefaultClaimsRetrieve
                             </div>
                         </div>
                     </div>
+                    <div class="mb-config">

```

Comment:
Examples are missing as a result all the code elements do not have the "View Sample" option.

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/2008/commits/0bc93ce7ddf833390582d6f005f85927788a4f2d